console/billing/how-to/download-an-invoice.mdx

```diff
@@ -13,9 +13,10 @@ categories:
   - billing
 ---
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+
```

Comment:
I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?